docs/topics/gradle/gradle-kotlin-dsl.md

```diff
@@ -0,0 +1,48 @@
+[//]: # (title: Using the Gradle Kotlin DSL)
+
+The Gradle Kotlin DSL is a domain specific language to write build scripts in more convenient way.
+
+## Using the Gradle Kotlin DSL
+
+When using [Gradle Kotlin DSL](https://github.com/gradle/kotlin-dsl), apply Kotlin plugins using the `plugins { ... }` block.
+If you apply them with `apply { plugin(...) }` instead, you may encounter unresolved references to the extensions generated
+by Gradle Kotlin DSL. To resolve that, you can comment out the erroneous usages, run the Gradle task `kotlinDslAccessorsSnapshot`,
+then uncomment the usages back and rerun the build or reimport the project into the IDE.
```

Comment:
A kind reminder: this is not my content and I didn't touch it when moved:) Well, this is rather some kind of note, not sure we should expand it visually with the list.

Reply:
Sorry I didn't know you weren't the original author. We can still change it though 😄 What makes you not like the idea of using a numbered list? You think it's too few steps?

Reply:
😅 The initial huge Gradle page was written by lots of authors.
Now we're working on splitting this page because this page raises from release to release – our developers are actively working on the Kotlin Gradle Plugin (KGP). Also, this page is very popular, this is one more reason to make it better.
I just took the page and split it according to the proposal (I gave the link to it). My idea was to make as few changes as possible😅 to merge it fast and continue bringing changes (because now I have to do PRs into this separate branch). That's why I marked in the comments where is the new content.
Anyway, nothing scary happened and I understand that the version control system might have confused you. However, I thought that you understood that most of the content is not mine and that's why you told me that your review is a bit out of scope. We are working on improving this content, so I appreciate your help, though I supposed we will make this "review-all-the-content-attentively" part later when the initial split is merged.

Reply:
> What makes you not like the idea of using a numbered list? You think it's too few steps?

I answered [above](https://github.com/JetBrains/kotlin-web-site/pull/3225#discussion_r1000738813):
> Well, this is rather a some kind of note, not sure we should expand it visually with the list.

Let's postpone this particular enhancement discussion because I'm going to convince the team to throw this page😁 It was in that thread in Slack.